### PR TITLE
OSSM-2147 Initialize analysis client with the actual revision

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by jewertow — APPROVED** [review]

**Comment by jewertow** [comment]
@luksa are you going to fix it in istio/istio as well?

**Comment by luksa** [comment]
Are you able to reproduce this upstream?

File: pkg/config/analysis/incluster/controller.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func NewController(stop <-chan struct{}, rwConfigStore model.ConfigStoreControll
 			Remove(collections.K8SApiextensionsK8SIoV1Customresourcedefinitions)
 	}
 
-	store, err := crdclient.NewForSchemas(kubeClient, "default",
```
Comment (dgn) [inline pkg/config/analysis/incluster/controller.go:66]: this hard-coded revision definitely looks like upstream is affected, too. except if they only ever want this to work on the default revision?

**Review by dgn — APPROVED** [review]
good catch!

**Pull request merged**